composer.json

```diff
@@ -12,7 +12,7 @@
         }
     ],
     "require": {
-        "php"                   : ">=5.5",
+        "php"                   : "^5.5.9",
```

Comment:
Thanks @dzuelke  ;) I will try the first option, but I don't understand how it can work. It `ext-blackfire` is added to the `require` section, if someone does not have blackfire, it should fail, right?